File: src/main/java/ar/edu/um/haberes/rest/model/ConversionLog.java
/**
 * 
 */
package ar.edu.um.haberes.rest.model;

import java.io.Serializable;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * @author daniel
 *
 */
@Entity
@EqualsAndHashCode(callSuper=false)
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ConversionLog extends Auditable implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = -3590846421373405607L;

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long version;
	private Integer anho = 0;
	private Integer mes = 0;

}

File: src/main/java/um/haberes/core/kotlin/model/ConversionLog.kt
package um.haberes.core.kotlin.model

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table
data class ConversionLog(

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var version: Long? = null,
    var anho: Int = 0,
    val mes: Int = 0

) : Auditable()
